src/pages/layer-2.js

```diff
@@ -0,0 +1,1084 @@
+          ethIcon={getImage(data.ethHome)}
+        />
+      </PaddedContent>
+
+      <PaddedContent>
+        <h2>Tools to be effective on layer 2</h2>
+        <TwoColumnContent>
+          <Flex50>
+            <ProductList
+              category="Information"
+              content={toolsData.information}
+            />
+          </Flex50>
+          <Flex50>
+            <ProductList
+              category="Wallet managers"
+              content={toolsData.walletManagers}
+            />
+          </Flex50>
+        </TwoColumnContent>
+      </PaddedContent>
+
+      <PaddedContent>
+        <h2>FAQ</h2>
+        <ExpandableCard title="Why is there no 'official' Ethereum L2?">
+          <p>
+            Just as there is no “official” Ethereum client, there is no
+            “official” Ethereum layer 2. Multiple teams will implement their
+            version of a layer 2, and the ecosystem as a whole will benefit from
+            diversity in these clients as a decentralizing point. Much like we
+            have multiple Ethereum clients developed by multiple teams in order
+            to have diversity in the network, this too will be how layer 2s
+            develop in the future.
+          </p>
+        </ExpandableCard>
+        <ExpandableCard title="What is the difference between optimistic and zero-knowledge rollups?">
+          <p>
+            Main difference being, validity proofs run the computations and post
+            a proof, whereas fault proofs only run the computations when faults
+            is suspected and needs to be checked. At the moment, most zk-rollups
+            are application specific, in contrast with optimistic rollups which
+            have largely been generalizable.
+          </p>
+        </ExpandableCard>
+        <ExpandableCard title="Is scaling at layer 1 possible?">
+          <p>
+            Yes. Currently in the Ethereum roadmap there are plans for shard
+            chains. While these are in the roadmap, further scaling through
+            layer 2 networks is still necessary.{" "}
+            <Link to="/upgrades/shard-chains/">More info on sharding</Link>.
+          </p>
+        </ExpandableCard>
+        <ExpandableCard title="What are the risks with layer 2?">
+          <p>
+            Sequencers may go down, leading you to have to wait to access funds.
+            While there may be a waiting period, you do still have access to
+            your funds unlike alt-l1's or sidechains.
+          </p>
+          <p>
+            Bridges are in their early stages of development, and it is likely
+            that the optimal bridge design has not been discovered yet. There
+            have been{" "}
+            <Link to="https://rekt.news/wormhole-rekt/">
+              recent hacks of bridges
+            </Link>
+            . <Link to="/bridges/">More information on bridges</Link>.
+          </p>
+        </ExpandableCard>
```

Comment:
One question we had in the draft that I think is worth keeping is the 'Does Bitcoin have layer 2?' question. Maybe a nice SEO hit and allows us to compare Bitcoin vs Ethereum scaling a little.

Any thoughts?

Reply:
Im open to it, it just seemed a little odd being on the page in the end I thought as we don't really discuss Bitcoin. Feels like we would just be shoe horning it in to hit the SEO in my opinion, but its not a strongly held one. Do you think it would add to the page to have this here?

Reply:
I think having content in FAQs that isn't touched on the page itself is fine. In my eyes, FAQs are a good place to explore semi-valuable content that didn't fit well with the story the page was trying to tell. 

Maybe a better question (and definitely one I could see users having) after reading this page is: is layer 2 an Ethereum specific thing? And if we're answering that, might they also be interested in what other blockchains have L2s?

Reply:
I like that more generalized question better. Will come up with something for that.